contracts/SlashIndicator.sol

```diff
@@ -19,11 +19,8 @@ import "./lib/RLPEncode.sol";
 contract SlashIndicator is ISlashIndicator,System,IParamSubscriber, IApplication{
   using RLPEncode for *;
 
-  // TODO: revert to 50 and 150
-  // uint256 public constant MISDEMEANOR_THRESHOLD = 50;
-  // uint256 public constant FELONY_THRESHOLD = 150;
-  uint256 public constant MISDEMEANOR_THRESHOLD = 5;
-  uint256 public constant FELONY_THRESHOLD = 10;
+  uint256 public constant MISDEMEANOR_THRESHOLD = 50;
```

Comment:
if you revert this, how can we test in dev-net?

Reply:
The generate script will handle this